lib/output.js

```diff
@@ -146,11 +150,10 @@ function OutputStream(options) {
               case "\b": return "\\b";
               case "\f": return "\\f";
               case "\x0B": return options.screw_ie8 ? "\\v" : "\\x0B";
-              case "\u2028": return "\\u2028";
-              case "\u2029": return "\\u2029";
-              case "\ufeff": return "\\ufeff";
```

Comment:
Since the logic for these 3 unicode values were changed, can you add them to all the ascii.js tests? (And regexp.js test if applicable).